aait/backend/group-2A/Application/Profiles/MappingProfile.cs

```diff
@@ -14,16 +14,16 @@ public class MappingProfile : AutoMapper.Profile
         
         CreateMap<Post, PostDto>().ReverseMap();
         CreateMap<Post, UpdatePostDto>().ReverseMap();
-        CreateMap<Post, CreateUserDTO>().ReverseMap();
+        CreateMap<Post, CreatePostDto>().ReverseMap();
```

Comment:
if there is not scenario that you are going to map Entity to CreateEntityDto or UpdateEntityDto , map the creeateEntityDto or UpdateEntityDto to Entity. you don't need to reverseMap(). example
CreateMap<CreatePostDto,Post>();